pyproject.toml

```diff
@@ -8,7 +16,7 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [tool.towncrier]
-    package = "lsst-versions"
+    package = "lsst_versions"
```

Comment:
Why is this needed? The release notes refer to `lsst-versions`.

Reply:
Towncrier was unable to load the package as "lsst-versions" but it works with "lsst_versions".

I don't know if there's actually a process that runs towncrier over this repo, the prior art has updates written directly to the changelog.

Reply:
We do generally use towncrier but for this package the changes are so rare that we usually make a release immediately so towncrier isn't really helping much. If you look in the history you will see that there are news fragments that have been pulled into release notes using towncrier.